src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -131,8 +131,9 @@ ProcessReturnCode parse_arguments(
         std::vector<option::Option> buffer(stats.buffer_max);
         option::Parser parse(usage, argc, argv, &options[0], &buffer[0]);
 
-        if (parse.error())
+        if (parse.error() || parse.nonOptionsCount())
```

Comment:
Show different error msg in this case.
Check file https://github.com/eProsima/Fast-DDS/blob/71d986f16f1e6b8a2bb86a85c6c86d1f7f9532d5/tools/fds/server.cpp#L69 to get an idea how to do it.

Reply:
:+1: